practitioner-service/README.md

```diff
@@ -94,3 +94,30 @@ Sets the validation rules for practitioner's name attributes using regex.
 Note: In this iteration, the field names are hard-coded, and the service will validate the existence of all three attributes (prefix, given name and family name) in the configuration file. Adding more attributes to the json configuration file will not dynamically add them to the validation process.
 Empty or null validation regex is converted to a "validate-any" expression.
 The current json file allows up to 35 characters for any field, and specifies only "family name" as mandatory (minimum characters = 1).
+
+## Running the service
+
+### Local profile (for local debugging)
+when running locally, the config server is not required. setup the following env vars and run the project:
+
+```sh
+SPRING_CLOUD_CONFIG_PROFILE=local
```

Comment:
Is this important to set if the config server is offline?

Reply:
typo